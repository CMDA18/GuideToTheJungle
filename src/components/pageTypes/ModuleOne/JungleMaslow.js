// @flow
import React, { Component } from 'react'
import Helmet from 'react-helmet'
// import styled from 'styled-components'
import Grid from '../../Grid'
import PageTitle from '../../PageTitle'
import Intro from '../../Intro'
import { jungleMaslow } from '../../../constants/content/index'
import TextAndImage from '../../TextAndImage'

type Props = {
  test: string
}

class JungleMaslow extends Component<Props> {
  render () {
    return (
      <div className='jungleMinds' data-test={this.props.test}>
        <Helmet
          title='J ungleMinds'
        />
        <Grid page='jungleMinds'>
          <header>
            <PageTitle {...jungleMaslow}/>
            <Intro {...jungleMaslow}/>
            <TextAndImage {...jungleMaslow}/>
          </header>
        </Grid>
      </div>
    )
  };
}

export default JungleMaslow
