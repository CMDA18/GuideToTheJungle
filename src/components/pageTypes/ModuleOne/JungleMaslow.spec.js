import React from 'react'
import { shallow } from 'enzyme'
import JungleMaslow from './JungleMaslow'

it('renders without crashing', () => {
  expect(shallow(<JungleMaslow />)).toMatchSnapshot()
})
