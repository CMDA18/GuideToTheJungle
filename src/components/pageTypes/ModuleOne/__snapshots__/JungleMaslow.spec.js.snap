// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <JungleMaslow />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
          title="J ungleMinds"
        />,
        <Grid
          page="jungleMinds"
        >
          <header>
            <PageTitle
              image="./images/team_design.jpg"
              introText="test"
              introTitle="Onze structuur"
              mediaCaption="test"
              pageTitle="1.2 Jungle Maslow"
              textMedia={
                Array [
                  Object {
                    "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                    "title": "Financiële autonomie",
                  },
                  Object {
                    "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                    "title": "Een ijzersterk team op een fijne plek ",
                  },
                ]
              }
            />
            <Intro
              image="./images/team_design.jpg"
              introText="test"
              introTitle="Onze structuur"
              mediaCaption="test"
              pageTitle="1.2 Jungle Maslow"
              textMedia={
                Array [
                  Object {
                    "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                    "title": "Financiële autonomie",
                  },
                  Object {
                    "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                    "title": "Een ijzersterk team op een fijne plek ",
                  },
                ]
              }
            />
            <TextAndImage
              image="./images/team_design.jpg"
              introText="test"
              introTitle="Onze structuur"
              mediaCaption="test"
              pageTitle="1.2 Jungle Maslow"
              textMedia={
                Array [
                  Object {
                    "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                    "title": "Financiële autonomie",
                  },
                  Object {
                    "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                    "title": "Een ijzersterk team op een fijne plek ",
                  },
                ]
              }
            />
          </header>
        </Grid>,
      ],
      "className": "jungleMinds",
      "data-test": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "defer": true,
          "encodeSpecialCharacters": true,
          "title": "J ungleMinds",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <header>
            <PageTitle
              image="./images/team_design.jpg"
              introText="test"
              introTitle="Onze structuur"
              mediaCaption="test"
              pageTitle="1.2 Jungle Maslow"
              textMedia={
                Array [
                  Object {
                    "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                    "title": "Financiële autonomie",
                  },
                  Object {
                    "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                    "title": "Een ijzersterk team op een fijne plek ",
                  },
                ]
              }
            />
            <Intro
              image="./images/team_design.jpg"
              introText="test"
              introTitle="Onze structuur"
              mediaCaption="test"
              pageTitle="1.2 Jungle Maslow"
              textMedia={
                Array [
                  Object {
                    "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                    "title": "Financiële autonomie",
                  },
                  Object {
                    "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                    "title": "Een ijzersterk team op een fijne plek ",
                  },
                ]
              }
            />
            <TextAndImage
              image="./images/team_design.jpg"
              introText="test"
              introTitle="Onze structuur"
              mediaCaption="test"
              pageTitle="1.2 Jungle Maslow"
              textMedia={
                Array [
                  Object {
                    "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                    "title": "Financiële autonomie",
                  },
                  Object {
                    "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                    "title": "Een ijzersterk team op een fijne plek ",
                  },
                ]
              }
            />
          </header>,
          "page": "jungleMinds",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <PageTitle
                image="./images/team_design.jpg"
                introText="test"
                introTitle="Onze structuur"
                mediaCaption="test"
                pageTitle="1.2 Jungle Maslow"
                textMedia={
                  Array [
                    Object {
                      "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                      "title": "Financiële autonomie",
                    },
                    Object {
                      "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                      "title": "Een ijzersterk team op een fijne plek ",
                    },
                  ]
                }
              />,
              <Intro
                image="./images/team_design.jpg"
                introText="test"
                introTitle="Onze structuur"
                mediaCaption="test"
                pageTitle="1.2 Jungle Maslow"
                textMedia={
                  Array [
                    Object {
                      "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                      "title": "Financiële autonomie",
                    },
                    Object {
                      "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                      "title": "Een ijzersterk team op een fijne plek ",
                    },
                  ]
                }
              />,
              <TextAndImage
                image="./images/team_design.jpg"
                introText="test"
                introTitle="Onze structuur"
                mediaCaption="test"
                pageTitle="1.2 Jungle Maslow"
                textMedia={
                  Array [
                    Object {
                      "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                      "title": "Financiële autonomie",
                    },
                    Object {
                      "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                      "title": "Een ijzersterk team op een fijne plek ",
                    },
                  ]
                }
              />,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "image": "./images/team_design.jpg",
                "introText": "test",
                "introTitle": "Onze structuur",
                "mediaCaption": "test",
                "pageTitle": "1.2 Jungle Maslow",
                "textMedia": Array [
                  Object {
                    "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                    "title": "Financiële autonomie",
                  },
                  Object {
                    "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                    "title": "Een ijzersterk team op een fijne plek ",
                  },
                ],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "image": "./images/team_design.jpg",
                "introText": "test",
                "introTitle": "Onze structuur",
                "mediaCaption": "test",
                "pageTitle": "1.2 Jungle Maslow",
                "textMedia": Array [
                  Object {
                    "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                    "title": "Financiële autonomie",
                  },
                  Object {
                    "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                    "title": "Een ijzersterk team op een fijne plek ",
                  },
                ],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "image": "./images/team_design.jpg",
                "introText": "test",
                "introTitle": "Onze structuur",
                "mediaCaption": "test",
                "pageTitle": "1.2 Jungle Maslow",
                "textMedia": Array [
                  Object {
                    "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                    "title": "Financiële autonomie",
                  },
                  Object {
                    "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                    "title": "Een ijzersterk team op een fijne plek ",
                  },
                ],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "header",
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defer={true}
            encodeSpecialCharacters={true}
            title="J ungleMinds"
          />,
          <Grid
            page="jungleMinds"
          >
            <header>
              <PageTitle
                image="./images/team_design.jpg"
                introText="test"
                introTitle="Onze structuur"
                mediaCaption="test"
                pageTitle="1.2 Jungle Maslow"
                textMedia={
                  Array [
                    Object {
                      "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                      "title": "Financiële autonomie",
                    },
                    Object {
                      "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                      "title": "Een ijzersterk team op een fijne plek ",
                    },
                  ]
                }
              />
              <Intro
                image="./images/team_design.jpg"
                introText="test"
                introTitle="Onze structuur"
                mediaCaption="test"
                pageTitle="1.2 Jungle Maslow"
                textMedia={
                  Array [
                    Object {
                      "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                      "title": "Financiële autonomie",
                    },
                    Object {
                      "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                      "title": "Een ijzersterk team op een fijne plek ",
                    },
                  ]
                }
              />
              <TextAndImage
                image="./images/team_design.jpg"
                introText="test"
                introTitle="Onze structuur"
                mediaCaption="test"
                pageTitle="1.2 Jungle Maslow"
                textMedia={
                  Array [
                    Object {
                      "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                      "title": "Financiële autonomie",
                    },
                    Object {
                      "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                      "title": "Een ijzersterk team op een fijne plek ",
                    },
                  ]
                }
              />
            </header>
          </Grid>,
        ],
        "className": "jungleMinds",
        "data-test": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "defer": true,
            "encodeSpecialCharacters": true,
            "title": "J ungleMinds",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <header>
              <PageTitle
                image="./images/team_design.jpg"
                introText="test"
                introTitle="Onze structuur"
                mediaCaption="test"
                pageTitle="1.2 Jungle Maslow"
                textMedia={
                  Array [
                    Object {
                      "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                      "title": "Financiële autonomie",
                    },
                    Object {
                      "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                      "title": "Een ijzersterk team op een fijne plek ",
                    },
                  ]
                }
              />
              <Intro
                image="./images/team_design.jpg"
                introText="test"
                introTitle="Onze structuur"
                mediaCaption="test"
                pageTitle="1.2 Jungle Maslow"
                textMedia={
                  Array [
                    Object {
                      "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                      "title": "Financiële autonomie",
                    },
                    Object {
                      "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                      "title": "Een ijzersterk team op een fijne plek ",
                    },
                  ]
                }
              />
              <TextAndImage
                image="./images/team_design.jpg"
                introText="test"
                introTitle="Onze structuur"
                mediaCaption="test"
                pageTitle="1.2 Jungle Maslow"
                textMedia={
                  Array [
                    Object {
                      "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                      "title": "Financiële autonomie",
                    },
                    Object {
                      "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                      "title": "Een ijzersterk team op een fijne plek ",
                    },
                  ]
                }
              />
            </header>,
            "page": "jungleMinds",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <PageTitle
                  image="./images/team_design.jpg"
                  introText="test"
                  introTitle="Onze structuur"
                  mediaCaption="test"
                  pageTitle="1.2 Jungle Maslow"
                  textMedia={
                    Array [
                      Object {
                        "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                        "title": "Financiële autonomie",
                      },
                      Object {
                        "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                        "title": "Een ijzersterk team op een fijne plek ",
                      },
                    ]
                  }
                />,
                <Intro
                  image="./images/team_design.jpg"
                  introText="test"
                  introTitle="Onze structuur"
                  mediaCaption="test"
                  pageTitle="1.2 Jungle Maslow"
                  textMedia={
                    Array [
                      Object {
                        "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                        "title": "Financiële autonomie",
                      },
                      Object {
                        "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                        "title": "Een ijzersterk team op een fijne plek ",
                      },
                    ]
                  }
                />,
                <TextAndImage
                  image="./images/team_design.jpg"
                  introText="test"
                  introTitle="Onze structuur"
                  mediaCaption="test"
                  pageTitle="1.2 Jungle Maslow"
                  textMedia={
                    Array [
                      Object {
                        "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                        "title": "Financiële autonomie",
                      },
                      Object {
                        "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                        "title": "Een ijzersterk team op een fijne plek ",
                      },
                    ]
                  }
                />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "image": "./images/team_design.jpg",
                  "introText": "test",
                  "introTitle": "Onze structuur",
                  "mediaCaption": "test",
                  "pageTitle": "1.2 Jungle Maslow",
                  "textMedia": Array [
                    Object {
                      "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                      "title": "Financiële autonomie",
                    },
                    Object {
                      "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                      "title": "Een ijzersterk team op een fijne plek ",
                    },
                  ],
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "image": "./images/team_design.jpg",
                  "introText": "test",
                  "introTitle": "Onze structuur",
                  "mediaCaption": "test",
                  "pageTitle": "1.2 Jungle Maslow",
                  "textMedia": Array [
                    Object {
                      "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                      "title": "Financiële autonomie",
                    },
                    Object {
                      "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                      "title": "Een ijzersterk team op een fijne plek ",
                    },
                  ],
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "image": "./images/team_design.jpg",
                  "introText": "test",
                  "introTitle": "Onze structuur",
                  "mediaCaption": "test",
                  "pageTitle": "1.2 Jungle Maslow",
                  "textMedia": Array [
                    Object {
                      "text": "Jungle Minds is gewoon van Jungle Minds en zo willen we het graag houden. Dus geen externe aandeelhouders en verborgen belangen. Want onze onafhankelijkheid geeft ons rust en vooral de ruimte om onze eigen creatieve ambitie na te jagen en soms ook gewoon ergens nee tegen te zeggen.",
                      "title": "Financiële autonomie",
                    },
                    Object {
                      "text": "Goede mensen zoeken goede mensen om mee te werken. En dus leggen we de lat hoog, hechten we veel waarde aan cultuur en investeren we flink in de persoonlijk ontwikkeling van onze mensen.",
                      "title": "Een ijzersterk team op een fijne plek ",
                    },
                  ],
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "header",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
